sql/parser/select.go

```diff
@@ -1,65 +1,70 @@
 package parser
 
 import (
-	"github.com/genjidb/genji/sql/query"
+	"fmt"
+
+	"github.com/genjidb/genji/sql/planner"
 	"github.com/genjidb/genji/sql/query/expr"
 	"github.com/genjidb/genji/sql/scanner"
 )
 
 // parseSelectStatement parses a select string and returns a Statement AST object.
 // This function assumes the SELECT token has already been consumed.
-func (p *Parser) parseSelectStatement() (query.SelectStmt, error) {
-	var stmt query.SelectStmt
+func (p *Parser) parseSelectStatement() (*planner.Tree, error) {
+	var cfg selectConfig
 	var err error
 
 	// Parse field list or query.Wildcard
-	stmt.Selectors, err = p.parseResultFields()
+	cfg.ProjectionExprs, err = p.parseResultFields()
 	if err != nil {
-		return stmt, err
+		return nil, err
 	}
 
 	// Parse "FROM".
 	var found bool
-	stmt.TableName, found, err = p.parseFrom()
-	if err != nil || !found {
-		return stmt, err
+	cfg.TableName, found, err = p.parseFrom()
+	if err != nil {
+		return nil, err
+	}
+	if !found {
```

Comment:
If the table is not found, shouldn't we return an error instead?

Reply:
No because we can do this: `SELECT 1 + 1`

Reply:
Why could we do that?

Reply:
This is standard, you can try with sqlite3 for example. This is a way to execute an expression and get the result